rosbag2_tests/resources/cdr_test/metadata.yaml

```diff
@@ -19,3 +19,5 @@ rosbag2_bagfile_information:
         type: test_msgs/Arrays
         serialization_format: cdr
       message_count: 4
+  compression_format: ""
```

Comment:
Do you need a test resource that does not contain the compression metadata keys (test backwards compatibility)?